### PR TITLE
docs: N8N Qdrant blog

### PR DESCRIPTION
**Pull request by Anush008**
This PR adds a blog about building a GitHub -> Qdrant workflow using the new Qdrant N8N node.

[Preview](https://deploy-preview-506--condescending-goldwasser-91acf0.netlify.app/blog/qdrant-n8n/)

### CODE REVIEW

File: qdrant-landing/content/blog/qdrant-n8n.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Our workflow will consist of two main parts. Refer to the [n8n quick start guide](https://docs.n8n.io/workflows/create/) to get acquainted with workflow semantics.
+
+- A workflow to ingest a GitHub repository into Qdrant
+- A workflow for a chat service with the ingested documents
+
+#### Workflow 1: GitHub Repository Ingestion into Qdrant
+
+![GitHub to Qdrant workflow](/blog/qdrant-n8n/load-demo.gif)
+
+For this workflow, we'll use the following nodes.
```
Comment (mjang) [inline qdrant-landing/content/blog/qdrant-n8n.md:37]:
```suggestion
For this workflow, we'll use the following nodes:
```

File: qdrant-landing/content/blog/qdrant-n8n.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- [Recursive Character Text Splitter](https://docs.n8n.io/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.textsplitterrecursivecharactertextsplitter/): Configure the [text splitter options](https://docs.n8n.io/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.textsplitterrecursivecharactertextsplitter/#node-parameters ). We use the defaults in this example.
+
+Connect the workflow to a manual trigger. Click "Test Workflow" to run it. You should be able to see the progress in real-time as the data is fetched from GitHub, transformed into vectors and loaded into Qdrant.
+
+#### Workflow 2: Chat Service with Ingested Documents
+
+![Chat workflow](/blog/qdrant-n8n/chat.png)
+
+The workflow use the following nodes
```
Comment (mjang) [inline qdrant-landing/content/blog/qdrant-n8n.md:53]:
```suggestion
The workflow use the following nodes:
```

File: qdrant-landing/content/blog/qdrant-n8n.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Once configured, hit the "Chat" button to initiate the chat interface and begin a conversation with your codebase.
+
+![Chat demo](/blog/qdrant-n8n/chat-demo.png)
+
+To embed the chat in your applications, consider using the [@n8n/chat](https://www.npmjs.com/package/@n8n/chat) package. Additionally, N8N supports scheduled workflows and can be triggered by events across various applications.
+
+## Further reading
+
+- [N8N Reference](https://docs.n8n.io/)
+- [Qdrant Node documentation](https://docs.n8n.io/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreqdrant/#qdrant-vector-store)
```
Comment (mjang) [inline qdrant-landing/content/blog/qdrant-n8n.md:72]:
```suggestion
- [n8n Documentation](https://docs.n8n.io/)
- [n8n Qdrant Node documentation](https://docs.n8n.io/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreqdrant/#qdrant-vector-store)
```

**Review by mjang — CHANGES_REQUESTED** [review]
Your content is easy to read. It's at a "grade level 8" on the Flesch-Kincaid scale, which is impressive for AI!

While I've checked the wording and basic technical content, I have _not_ run your examples (too much to do)

File: qdrant-landing/content/blog/qdrant-n8n.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- A workflow to ingest a GitHub repository into Qdrant
+- A workflow for a chat service with the ingested documents
+
+#### Workflow 1: GitHub Repository Ingestion into Qdrant
+
+![GitHub to Qdrant workflow](/blog/qdrant-n8n/load-demo.gif)
+
+For this workflow, we'll use the following nodes.
+
+- [Qdrant Vector Store - Insert](https://docs.n8n.io/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreqdrant/#insert-documents): Configure with API key and collection name. If the collection doesn't exist, it's automatically created with the appropriate configurations.
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/blog/qdrant-n8n.md]: https://github.com/qdrant/landing_page/pull/506/commits/7d75f005f6eaafc9103d679786842766fd44c857

File: qdrant-landing/static/blog/qdrant-n8n/preview/social_preview.jpg
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/static/blog/qdrant-n8n/preview/social_preview.jpg:1]: It is from the n8n integrations page(auto-generated for the all the integrations). So unfortunately, we can't have that updated.

**Review by mjang — APPROVED** [review]
LGTM from a TW PoV. Thanks @Anush008 !

**Pull request merged**